ios/FlankerNativeComponents/GameManager.swift

```diff
@@ -191,7 +191,11 @@ class GameManager {
 
   func checkedAnswer(button: SelectedButton) {
     invalidateTimers()
-    delegate?.setEnableButton(isEnable: false)
+
+    DispatchQueue.main.async {
+      self.delegate?.setEnableButton(isEnable: false)
```

Comment:
Since this is a delegate function you can wrap the `setEnableButton` function in FlankerView.swift instead of here to avoid repeating this multiple times.

Same goes for `updateText`.